backend/census_historical_migration/workbooklib/findings_text.py

```diff
@@ -19,29 +20,43 @@
 
 mappings = [
     SheetFieldMap(
-        "reference_number", "findingrefnums", "finding_ref_number", None, str
+        "reference_number", "FINDINGREFNUMS", "finding_ref_number", None, str
     ),
-    SheetFieldMap("text_of_finding", "text", "finding_text", None, str),
+    SheetFieldMap("text_of_finding", "TEXT", "finding_text", None, str),
     SheetFieldMap(
-        "contains_chart_or_table", "chartstables", WorkbookFieldInDissem, None, str
+        "contains_chart_or_table", "CHARTSTABLES", WorkbookFieldInDissem, None, str
     ),
 ]
 
 
+def _get_findings_text(dbkey):
+    return FindingsText.objects.filter(DBKEY=dbkey).order_by("SEQ_NUMBER")
+
+
 def generate_findings_text(dbkey, year, outfile):
+    """
+    Generates a findings text workbook for a given dbkey.
+
+    Note: This function assumes that all the findings text
+    information in the database is related to the same year.
+    """
     logger.info(f"--- generate findings text {dbkey} {year} ---")
-    Gen = dynamic_import("Gen", year)
-    Findingstext = dynamic_import("Findingstext", year)
+
     wb = pyxl.load_workbook(sections_to_template_paths[FORM_SECTIONS.FINDINGS_TEXT])
 
-    g = set_uei(Gen, wb, dbkey)
+    audit_header = get_audit_header(dbkey)
+
+    set_workbook_uei(wb, audit_header.UEI)
+
+    findings_texts = _get_findings_text(dbkey)
```

Comment:
I would make `findings_texts` and `_get_findings_text()` either both singular or both plural